<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_tar</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>source folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_source">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushsource">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>target folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_target">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushtarget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRun">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushsource</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slotsource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushtarget</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slottarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRun</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slotRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotsource()</slot>
  <slot>slottarget()</slot>
  <slot>slotRun()</slot>
 </slots>
</ui>
